docker-compose.yml

```diff
@@ -133,6 +150,60 @@ services:
      env_file:
        - ./notification/local.env
      restart: on-failure
+
+  filebeat:
+     hostname: filebeat
+     build:
+      context: ../logging/filebeat
+      dockerfile: Dockerfile

```

Comment:
Actually if the 3 configurations are in 3 different folders, the "dockerfile" param is not needed